crates/bevy_render/src/camera/projection.rs

```diff
@@ -161,17 +161,49 @@ pub enum ScalingMode {
     FixedHorizontal(f32),
 }
 
+/// Project a 3D space onto a 2D surface using parallel lines, i.e., objects have the same
+/// apparent size regardless of depth. The volume contained in the projection is called the view frustum.
```

Comment:
We should say that the volume is a rectangular prism, so that the `top`, `bottom`, `left`, `right`, `near`, and `far` make sense to the reader in this context.

Reply:
I did consider this, but "parallel lines" is more accurate to the technical definition of orthographic projection.

Maybe something along the lines of "Since the viewport is rectangular, The view frustum is in the shape of a rectangular prism" at the end of the paragraph would work?